file_upload/app.py

```
@@ -0,0 +1,113 @@
+            save_path = os.path.join(app.config['UPLOAD_FOLDER'], filename)
+            file.save(save_path)
+
+            # Save to DB
+            new_upload = Upload(filename=filename)
+            db.session.add(new_upload)
+            db.session.commit()
+
+            return jsonify({'message': f'File {filename} uploaded successfully'})
+        else:
+            return jsonify({'error': 'File type not allowed'}), 400
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)